pkgs/halo2-wasm-ext/README.md

```diff
@@ -0,0 +1,3 @@
+# `halo2-wasm-ext`
+
+ECDSA circuit built with [axiom-crypto/halo2-lib](https://github.com/axiom-crypto/halo2-lib).
```

Comment:
can you add a bit more info about this crate being to override some stuff in `halo2-wasm`. And why it is necessary/useful to do so?

Reply:
Done.